UnityProject/Assets/Scripts/Util/Independent/FluentRichText/StringExtensions.cs

```diff
@@ -38,5 +38,17 @@ public static class StringExtensions
 		public static string Superscript(this string text) => new SuperscriptStrategy().ApplyStyle(text);
 		public static string VerticalOffset(this string text, string amount) => new VerticalOffsetStrategy(amount).ApplyStyle(text);
 		public static string Width(this string text, string amount) => new WidthStrategy(amount).ApplyStyle(text);
+
+		public static string ToUpperFirstLetter(this string source)
```

Comment:
there is an extension method that does this already, if I'm not mistaken it is ``.Capitalize()``